data/xml/2020.cl.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2020.cl">
+  <volume id="1">
+    <meta>
+      <booktitle>Computational Linguistics, Volume 46, Issue 1 - March 2020</booktitle>
+      <month>March</month>
+      <year>2020</year>
+    </meta>
+
+    <paper id="1">
```

Comment:
Nitpicky but we typically don't have these spaces between elements. I think if you call `anthology.util.indent(tree.getroot())` it will fix it.